lib/smart_proxy_dns_infoblox/plugin_configuration.rb

```diff
@@ -12,7 +12,7 @@ def load_dependency_injection_wirings(container_instance, settings)
                                       ::Infoblox.wapi_version = '2.0'
                                       ::Infoblox::Connection.new(:username => settings[:username],
                                                                  :password => settings[:password],
-                                                                 :host => settings[:dns_server],
+                                                                 :host => settings[:infoblox_host],
```

Comment:
I think ```:infoblox_host``` should be an optional parameter and only used when set. In the absence of ```:infoblox_host```, the value in ```:dns_server``` setting should be used.